### DISCUSSION TITLE
Why sonar cloud cannot get code converage report?

### DISCUSSION THREAD

**Discussion by mgsky1** [apache/dolphinscheduler#8300] [Q&A]
I use PowerMockito for testing my code. In IDEA, it runs successfully and the code has 63% covered in methods and 79% in lines, but in sonar, the coverage is 0%.

I notice this problem has been solved in [SOC2021](https://summer.iscas.ac.cn/#/org/prodetail/210290131?lang=en). Why this problem raise again?

<img width="1427" alt="QQ20220208-002258@2x" src="https://user-images.githubusercontent.com/23556693/152828747-316dd2a1-587e-426c-b0bc-c1c0d897b4c5.png">

<img width="1382" alt="QQ20220208-002401@2x" src="https://user-images.githubusercontent.com/23556693/152828941-c9f50237-2cf6-41bf-93f9-67a09ae51c0c.png">

The testing code refers to #6272 

@CalvinKirs

**Comment by shink** [comment]
Hi, has this issue been resolved?
